qcengine/programs/tests/test_programs.py

```diff
@@ -343,3 +343,42 @@ def test_openmm_cmiles_gradient_nomatch():
         "molecule.add_conformer given input of the wrong shape: Given (3, 3), expected (8, 3)"
         in ret.error.error_message
     )
+
+
```

Comment:
I'm not able to get success on this test even when rolling with latest `openmmforcefields` master. I'd like to understand what's happening here before we merge, or at least reproduce success.

Reply:
Was able to get success after a `pip install .` instead of a `pip install -e .` for `openmmforcefields` (raised [this](https://github.com/openmm/openmmforcefields/pull/124) for the future). Can verify this works!

Reply:
I also see that we can't really expect e.g. `openff-1.0.0` forcefields to work yet until OpenMM 0.7.5 comes out. I'm satisfied that this is ready for merge!

Reply:
@dotsdl not sure what you mean here? We should see all smirnoff99frost and openforcefields work now with OpenMM7.4.2, [this](https://github.com/MolSSI/QCEngine/blob/a45fe16b48157e52a86dd3ec7fa40746726feb42/qcengine/programs/tests/test_programs.py#L300) test uses `openff-1.0.0` and should be successful.

Reply:
Oh man...I am losing it. Thanks @jthorton! My eyes have glazed over it at this point. :facepalm:

Reply:
no problem this has been a long one!